tests/legacy-cli/e2e/tests/basic/update-7.0.ts

```diff
@@ -0,0 +1,39 @@
+import { createProjectFromAsset } from '../../utils/assets';
+import { ng, silentNpm } from '../../utils/process';
+import {
+  isPrereleaseCli, removeHttpDep, useBuiltPackages, useCIChrome, useCIDefaults,
+} from '../../utils/project';
+
+
+export default async function () {
+  const extraUpdateArgs = await isPrereleaseCli() ? ['--next', '--force'] : [];
+
+  // Create new project from previous version files.
+  // We must use the original NPM packages to force a real update.
+  // This also means that we can only truly test updates after a release.
+  await createProjectFromAsset('7.0-project', true);
+  // Update the CLI.
+  await ng('update', '@angular/cli', ...extraUpdateArgs);
```

Comment:
@clydin is there any good way of testing updates without waiting for a release?

Reply:
Not the entire update.  The CLI migrations can be tested by manually installing the package and then running update with the migrate only option.